packages/fxa-auth-server/lib/senders/templates/partials/appBadges.html

```diff
@@ -0,0 +1,18 @@
+<tr style="page-break-before: always">
+  <td align="center" style="padding: 40px 0 10px;">
+    <a href="{{{iosUrl}}}">
+      <img src="https://accounts.firefox.com/images/apple_app_store_button/en.svg" width="152" height="45" alt="" style="-ms-interpolation-mode: bicubic;" />
+    </a>
+    <a href="{{{androidUrl}}}">
+      <img src="https://accounts.firefox.com/images/google_play_store_button/en@2x.png" width="152" height="45" alt="" style="-ms-interpolation-mode: bicubic;" />
```

Comment:
Because these images are the only content of the link it would probably be good to give them alt tag values